packages/mobile/src/screens/edit-track-screen/EditTrackModalScreen.tsx

```diff
@@ -51,7 +51,10 @@ export const EditTrackModalScreen = () => {
       trackImage && isImageUriSource(trackImage.source)
         ? trackImage.source.uri
         : undefined,
-    isUpload: false
+    isUpload: false,
+    isCover:
+      track.cover_original_artist !== null &&
```

Comment:
Just confirming that `isCover` with empty strings is a valid state?

Reply:
yup!